### ISSUE TITLE
Bug: Postgres expression index with multiple expressions runs correct SQL, but produces incorrect schema modifications

### ISSUE THREAD

**Issue by fj** [SchemaPlus/schema_plus#225]
Running this migration

```
class TestingMigration < ActiveRecord::Migration
  def change
    change_table :foo do |t|
      t.index(
        expression: "bar, (baz->>'wizzle')",
        name: 'idx_test'
      )
    end
  end
end
```

will run this SQL command

```
CREATE  INDEX  "idx_test" ON "foo"  (bar, (baz->>'wizzle'))
```

but will add this incomplete directive to the schema:

```
 # expression was dropped!
 add_index "foo", ["bar"], name: "idx_test"
```

The expected result is that we instead get something like

```
 add_index "foo", ["bar", "(baz->>'wizzle')"], name: "idx_test"
```

in the schema.

**Comment by ronen** [comment]
Closed in favor of SchemaPlus/schema_plus_pg_indexes#5

**Comment by ronen** [comment]
@fj FYI this should now be fixed in schema_plus_pg_indexes v0.1.7
